tests/storage/cdi_upload/conftest.py

```
@@ -119,3 +118,31 @@ def primary_udn_for_upload(admin_client, udn_namespace_for_dv_upload):
             status=udn.Condition.Status.TRUE,
         )
         yield udn
+
+
+@pytest.fixture()
+def uploaded_windows_dv(
```

Flake8 (7.3.0)

[error] redefinition of unused 'uploaded_windows_dv' from line 74

(F811)